pkg/vm/memory/relocatable_test.go

```diff
@@ -8,6 +8,50 @@ import (
 	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
 )
 
+func TestMaybeRelocatableIsEqual(t *testing.T) {
+	rel_felt_one := memory.NewMaybeRelocatableFelt(lambdaworks.FeltFromUint64(24))
+	rel_felt_two := memory.NewMaybeRelocatableFelt(lambdaworks.FeltFromUint64(24))
+
+	rel_rel_one := memory.NewMaybeRelocatableRelocatable(memory.Relocatable{2, 4})
+	rel_rel_two := memory.NewMaybeRelocatableRelocatable(memory.Relocatable{2, 4})
+
+	rel_felts_are_equal := rel_felt_one.IsEqual(rel_felt_two)
+	rel_rel_are_equal := rel_rel_one.IsEqual(rel_rel_two)
+	rel_rel_and_rel_felt_are_equal := rel_felt_one.IsEqual(rel_rel_one)
+
+	if !rel_felts_are_equal {
+		t.Errorf("MaybeRelocatable(0) should equal MaybeRelocatable(1)")
+	}
+	if !rel_rel_are_equal {
+		t.Errorf("MaybeRelocatable(2) should equal MaybeRelocatable(3)")
+	}
+	if rel_rel_and_rel_felt_are_equal {
+		t.Errorf("MaybeRelocatable(0) should equal MaybeRelocatable(2)")
+	}
```

Comment:
`MaybeRelocatable(i)` seems a bit confusing because that name doesn't appear somewhere else in the test. Maybe you can change it to something like:
```
	if !rel_felts_are_equal {
		t.Errorf("%s and %s are not equal", rel_felt_one, rel_felt_two)
	}
	if !rel_rel_are_equal {
		t.Errorf("%s and %s are not equal", rel_rel_one, rel_rel_two)
	}
	if rel_rel_and_rel_felt_are_equal {
		t.Errorf("%s and %s are not equal", rel_felt_one, rel_rel_one)
	}
```

Reply:
@gabrielbosio updated and rebased
removed redundant check for rel_rel_and_rel_fel_are_equal in IsEqual test when its checked in IsNotEqual test